api/topology/src/main/java/quarks/topology/TStream.java

```diff
@@ -4,18 +4,15 @@
 */
 package quarks.topology;
 
+import quarks.function.*;
```

Comment:
We don't have a formal coding style doc but the rest of the code uses explicit imports instead of wildcards.  Generally, it's best to avoid arbitrary changes when introducing new functionality.  Generally, nice work btw.